### PR TITLE
Add Lali Akhil Raj

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -173,6 +173,7 @@
 - [Arpit Mohan](https://dev.to/mohanarpit/i-am-a-yc-alumnus-3x-vc-funded-founder-cto-ama-4006) - Distributed Systems Engineer, Java and Golang developer, 3x CTO, [blogger](https://blog.arpitmohan.com).
 - [Yohix](https://github.com/yohix/ama) - iOS and web developer, open-source tool maker, loves coffee and music, he/him.
 - [Ittussarom Ynohtna](https://github.com/thetechrobo/ama) - Hobbyist Python 3.x developer who corrects grammar so much he loses friends.
+- [Lali Akhil Raj](https://github.com/Lalisfeed/ama) - Creator of [Lalisfeed](https://github.com/Lalisfeed), includes releases from Web to AI.
```
Comment (sindresorhus) [inline readme.md]: Your description doesn't make much sense.

Comment (ghost) [reply readme.md]: 😅 Is it okay for now? **I made few changes**.

File: readme.md [hunk 2]
```diff
@@ -173,7 +173,7 @@
 - [Arpit Mohan](https://dev.to/mohanarpit/i-am-a-yc-alumnus-3x-vc-funded-founder-cto-ama-4006) - Distributed Systems Engineer, Java and Golang developer, 3x CTO, [blogger](https://blog.arpitmohan.com).
 - [Yohix](https://github.com/yohix/ama) - iOS and web developer, open-source tool maker, loves coffee and music, he/him.
 - [Ittussarom Ynohtna](https://github.com/thetechrobo/ama) - Hobbyist Python 3.x developer who corrects grammar so much he loses friends.
-- [Lali Akhil Raj](https://github.com/Lalisfeed/ama) - Creator of [Lalisfeed](https://github.com/Lalisfeed), includes releases from Web to AI.
+- [Lali Akhil Raj](https://github.com/Lalisfeed/ama) - Creating Marvelous Stuff on Web & AI. View my works at [Lalisfeed](https://github.com/Lalisfeed).
```
Comment (ghost) [inline readme.md]: This ☝ one

File: readme.md [hunk 3]
```diff
@@ -173,7 +173,7 @@
 - [Arpit Mohan](https://dev.to/mohanarpit/i-am-a-yc-alumnus-3x-vc-funded-founder-cto-ama-4006) - Distributed Systems Engineer, Java and Golang developer, 3x CTO, [blogger](https://blog.arpitmohan.com).
 - [Yohix](https://github.com/yohix/ama) - iOS and web developer, open-source tool maker, loves coffee and music, he/him.
 - [Ittussarom Ynohtna](https://github.com/thetechrobo/ama) - Hobbyist Python 3.x developer who corrects grammar so much he loses friends.
-- [Lali Akhil Raj](https://github.com/Lalisfeed/ama) - Creating Marvelous Stuff on Web & AI. View my works at [Lalisfeed](https://github.com/Lalisfeed).
+- [Lali Akhil Raj](https://github.com/Lalisfeed/ama) - Creating stuff for web & AI.
```
Comment (ghost) [inline readme.md:176]: Oh, this is nice! _XD_

**Pull request merged**